editor/src/components/shared/project-components.ts

```diff
@@ -565,7 +565,7 @@ export const fragmentElementsDescriptors: ComponentDescriptorsForFile = {
 export const mapElementDescriptors: ComponentDescriptorsForFile = {
   map: {
     properties: {},
-    supportsChildren: false,
+    supportsChildren: true,
```

Comment:
since we do allow wrapping in a list now

Reply:
This can be problematic, because it enables insertion (e.g. the plus button in the navigator), even though we can not insert into a list.

Reply:
reverted it back to not supporting children, and replaced the logic to special-case the list element